galaxy/tests.py

```diff
@@ -139,7 +145,24 @@ def test_page_is_citizen(self):
         )
 
     def test_page_not_citizen(self):
-        Galaxy.rule()
+        galaxy = Galaxy.objects.create()
+        galaxy.rule(0)  # We want all users here
         self.client.login(username="root", password="plop")
         response = self.client.get("/galaxy/2/")
```

Comment:
I know this has not been modified in the current run, but while you are at it, could refactor this hardcoded url into a call to the django's `reverse` function ?

Reply:
Done